### PR TITLE
pnpm: add Indonesian translation

### PR DESCRIPTION
**Pull request by Paskalouis**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by kbdharun** [comment]
Welcome to tldr @Paskalouis, please sign the CLA in the above comment.

File: pages.id/common/pnpm.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+- Copot pemasangan paket dan hapus dari daftar dependensi di package.json:
+
+`pnpm remove {{module_name}}`
+
+- Mencetak pohon dependensi yang diinstal secara lokal:
+
+`pnpm list`
+
+- Buat daftar modul tingkat atas yang diinstal secara global:
```
Comment (kbdharun) [inline pages.id/common/pnpm.md:35]:
```suggestion
- Buat daftar modul tingkat atas yang diinstal secara [g]lobal:
```

File: pages.id/common/pnpm.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+- Unduh versi tertentu dari sebuah paket dan menambahkan ke daftar dependensi di package.json:
+
+`pnpm add {{nama_modul}}@{{versi}}`
+
+- Unduh paket dan menambahkan ke daftar dependensi dev di package.json:
+
+`pnpm add -D {{module_name}}`
+
+- Unduh paket dan instal secara global:
```
Comment (kbdharun) [inline pages.id/common/pnpm.md:23]:
```suggestion
- Unduh paket dan instal secara [g]lobal:
```

File: pages.id/common/pnpm.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+- Unduh semua paket yang terdaftar sebagai dependensi di package.json:
+
+`pnpm install`
+
+- Unduh versi tertentu dari sebuah paket dan menambahkan ke daftar dependensi di package.json:
+
+`pnpm add {{nama_modul}}@{{versi}}`
+
+- Unduh paket dan menambahkan ke daftar dependensi dev di package.json:
```
Comment (kbdharun) [inline pages.id/common/pnpm.md:19]:
```suggestion
- Unduh paket dan menambahkan ke daftar dependensi [D]ev di package.json:
```

**Review by kbdharun — APPROVED** [review]
Thanks for your contribution, the translations LGTM. I just have some suggestions for this page.

Tip: Try to use mnemonics in translated pages if possible if it is present on the main page too.

**Review by kbdharun — APPROVED** [review]

**Review by blueskyson — APPROVED** [review]

**Pull request merged**